File: Behavorial/Command/Command_Java/Television.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Command;

/**
 *
 * @author shantanu1395
 */

//Receiver
class Television implements ElectronicDevice{

    private int volume = 0;
    
    @Override
    public void on() {
        System.out.println("Television On");
    }

    @Override
    public void off() {
        System.out.println("Television Off");
    }

    @Override
    public void volumeUp() {
        volume++;
        System.out.println("Volume :"+volume);
    }

    @Override
    public void volumeDown() {
        volume--;
        System.out.println("Volume :"+volume);
    }
    
}

File: Behavorial/Command/Command_Python/Television.py
from ElectronicDevice import *

class Television(ElectronicDevice):

    def __init__(self):
        self.volume = 0
        self.name = 'Television'

    def on(self):
        print('TV On')

    def volumeUp(self):
        self.volume += 1
        print('Television Volume : ' + str(self.volume))

    def off(self):
        print('TV Off')

    def volumeDown(self):
        self.volume -= 1
        print('Television Volume : ' + str(self.volume))
